src/containers/Tenant/Diagnostics/Overview/TableInfo/TableInfo.tsx

```diff
@@ -17,27 +17,40 @@ const b = cn('ydb-diagnostics-table-info');
 
 interface TableInfoProps {
     data?: TEvDescribeSchemeResult;
+    type?: EPathType;
+    olapStats?: KeyValueRow[];
 }
 
-export const TableInfo = ({data}: TableInfoProps) => {
+export const TableInfo = ({data, type, olapStats}: TableInfoProps) => {
     const entityName = getEntityName(data?.PathDescription);
 
     const {
-        generalTableInfo = [],
+        generalInfo = [],
         tableStatsInfo = [],
         tabletMetricsInfo = [],
         partitionConfigInfo = [],
-    } = useMemo(() => prepareTableInfo(data), [data]);
+    } = useMemo(() => prepareTableInfo(data, type, olapStats), [data, type, olapStats]);
 
     return (
         <div className={b()}>
             <InfoViewer
-                info={generalTableInfo}
+                info={generalInfo}
                 title={entityName}
                 className={b('info-block')}
                 renderEmptyState={() => <div className={b('title')}>{entityName}</div>}
             />
             <div className={b('row')}>
+                <div className={b('col')}>
```

Comment:
I moved TableStats to the left to have similar view for common `Table`, `ColumnStore`, standalone `ColumnTable`, in-store `ColumnTable` and system `Table` (the last two don't have `TabletMetrics` or `PartitionConfig`, so they have `TableStats` on the left side